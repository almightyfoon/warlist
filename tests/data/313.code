c6501b8-9292939393937i7j7kg89u7FbFchchgi__5tc59vbH8Pe08idH9u73chch