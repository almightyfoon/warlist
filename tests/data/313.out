https://conflictchamber.com/#c6501b8-9292939393937i7j7kg89u7FbFchchiF__5tc59vbH8Pe08idH9u73chch

Mercenary Army - 75 / 75 points

[Gorten 1] Gorten Grundback [+31]
 - Grundback Blaster [6]
 - Grundback Blaster [6]
 - Grundback Gunner [6]
 - Grundback Gunner [6]
 - Grundback Gunner [6]
 - Grundback Gunner [6]
 - Reinholdt, Gobber Speculator [4]
Anastasia di Bray [3]
Gorman di Wulfe, Rogue Alchemist [4]
Major Harrison Gibbs [4]
Ragman [4]
Rorsh [15]
Wrong Eye [17]
Kayazy Eliminators [5]
Kayazy Eliminators [5]
Thorn Gun Mages [9]

Mercenary Army - 75 / 75 points

[MacBain 1] Drake MacBain [+30]
 - Galleon [39]
 - Sylys Wyshnalyrr, the Seeker [4]
Dahlia Hallyr [17]
Dirty Meg [3]
Gastone Crosse [4]
 - Buccaneer [6]
Gobber Tinker [2]
Ragman [4]
Croe's Cutthroats (max) [16]
Kayazy Eliminators [5]
Kayazy Eliminators [5]
